feat(docs): 添加第13期周刊内容

jinzhongjia opened the pull request:
- 在 mkdocs.yml 导航中添加第13期入口
- 创建 013.md 文件包含2025年5月26日至6月1日的Neovim周报
- 涵盖文档系统重组、LSP功能增强、Tree-sitter升级等开发近况
- 包含内置插件管理器提案、网络访问功能等社区资讯

jinzhongjia committed:
feat(docs): 添加第13期周刊内容
- 在 mkdocs.yml 导航中添加第13期入口
- 创建 013.md 文件包含2025年5月26日至6月1日的Neovim周报
- 涵盖文档系统重组、LSP功能增强、Tree-sitter升级等开发近况
- 包含内置插件管理器提案、网络访问功能等社区资讯

jinzhongjia committed:
docs(posts): 完善第13期周刊内容和格式

- 更新标题为"Neovim 中文周刊 第 13 期"
- 添加周刊介绍和版块说明
- 移除插件推荐部分的多余空行
- 新增"关于"版块，包含贡献者致谢和周刊信息
- 添加周刊网站、项目地址和RSS订阅链接
- 补充发布和修订日期信息

jinzhongjia commented:
你看着要是没问题可以直接发

v1nh1shungry commented on posts/013.md:
```diff
@@ -0,0 +1,147 @@
+
+本周 Neovim 社区依然活跃，多个重要功能提案和讨论引发广泛关注。以下是本周的社区亮点：
+
+### 内置插件管理器提案
+
+[PR #34009](https://github.com/neovim/neovim/pull/34009) 提出实现内置插件管理器 `vim.pack`，这是社区长期以来的期待。该提案由 [mini.nvim](https://github.com/echasnovski/mini.nvim) 的作者 [echasnovski](https://github.com/echasnovski) 提出，设计思路沿用了他的 [mini.deps](https://github.com/echasnovski/mini.deps) 插件。
+
+`vim.pack` 实现了一个 in-process LSP，用于为 `nvimpack` 提供目录等功能。根据当前设计，有两个功能明确不被考虑：
+
+- 位于 `pack/*/start` 的插件。所有由 `vim.pack` 管理的插件都会被放到 `$XDG_DATA_HOME/nvim/site/pack/core/opt`，这引起了一定的争议。
+- 插件懒加载。
+
+由于内置插件管理器一直是社区关注的焦点，echasnovski 恳请所有非 Nvim Team 成员的用户在仔细思考留言是否有帮助后再添加评论，以避免该 PR 有过多的留言。
+
+### 网络访问功能实现
+
+[PR #33964](https://github.com/neovim/neovim/pull/33964) 实现了 Lua 模块 `vim.net`，用于支持网络访问功能。目前该模块仅实现了一个 API `vim.net.download()`，用于从指定的 URL 下载文件。
+
+`vim.net` 可用于以下用途：
+
+- 替换 `:h netrw` 的前置任务。[PR #32280](https://github.com/neovim/neovim/pull/32280) 希望能够将 Netrw 替换为若干个独立的 Lua 模块，主要原因是 Netrw 包含了太多功能，这意味着用户为了使用 Netrw 的某一个功能就必须启用 Netrw 提供的所有功能，不符合 Neovim 的设计哲学。
+- 下载 `spellfile`（[Issue #7189](https://github.com/neovim/neovim/issues/7189)）
+- 访问远端文件（`:e ftp://foo.bar/baz`）
+- 下载 tree-sitter parser
+- 用于插件管理器下载其自身
+
+这是第三位尝试实现 `vim.net` 的开发者，前两次尝试（[PR #23461](https://github.com/neovim/neovim/pull/23461) 和 [PR #29104](https://github.com/neovim/neovim/pull/29104)）均未成功合并。当前实现的核心思路是一个使用 `vim.system()` 的 curl wrapper，特意维持该 PR 的实现最简化。
```
这两节与第 11 期重复了，报导的内容基本一致，甚至有几句话是一模一样的 :)

jinzhongjia replied:
新的 pack提案确实漏掉了

这些仍然加进来是因为本周对应的讨论有点进展

v1nh1shungry replied:
> 这些仍然加进来是因为本周对应的讨论有点进展

那或许应该着重于进展的新内容？不过这是一条出于我个人喜好的建议，你可以自由 resolve 掉。

jinzhongjia replied:
重复内容已经移除了

v1nh1shungry commented on posts/013.md:
```diff
@@ -0,0 +1,147 @@
+
+- [AI 导向的 LSP 功能](https://github.com/neovim/neovim/discussions/34254)：由 [ofseed](https://github.com/ofseed) 提出，旨在实现 AI 导向的 LSP 功能和 LSP 数据缓存模块。该项目将增强 Neovim 与 AI 工具的集成能力，并通过缓存机制提升 LSP 性能。
+
+### 发行版和插件管理器讨论
+
+社区中出现了关于 Neovim 发行版维护模式的讨论。[LazyVim](https://github.com/LazyVim/LazyVim) 是一个相当热门的 Neovim 发行版，然而长期以来，只有其维护者 [folke](https://github.com/folke) 拥有该仓库的 push 权限。这令一些用户感到担忧，并询问是否有由多人社区维护的发行版选择。
+
+多数用户推荐了 [NvChad](https://github.com/NvChad/NvChad) 和 [AstroNvim](https://github.com/AstroNvim/AstroNvim) 作为社区维护的替代选择。
+
+除了 LazyVim，folke 还持有众多热门 Neovim 插件，其中包括目前最流行的插件管理器 [lazy.nvim](https://github.com/folke/lazy.nvim)，这些项目的开发进度受到社区密切关注。
```
同上，第 11 期报导过一样的内容

jinzhongjia committed:
docs(posts): 更新第13期内容格式和链接标准化

- 统一Issue链接格式，为#34184添加"Issue"前缀
- 更新内置插件管理器章节，替换为saccarosium的新提案
- 删除vim.net网络访问功能和发行版讨论章节
- 移除v1nh1shungry贡献者致谢段落
- 修正日期格式标点符号

v1nh1shungry commented on posts/013.md:
```diff
@@ -0,0 +1,124 @@
+
+这些改进共同提升了外部 UI 的用户体验和一致性，对使用自定义 UI 的用户尤为重要。
+
+### LSP 功能增强
+
+LSP（语言服务器协议）相关功能获得了多项重要改进：
+
+1. 新增 `workspace/diagnostic` 方法支持 ([#34262](https://github.com/neovim/neovim/pull/34262))(尚未合并)：由 [@MariaSolOs](https://github.com/MariaSolOs) 实现，允许开发者查看整个工作区的诊断信息，而不仅限于已打开的文件
+2. 改进 `vim.lsp.log.set_format_func()` 以控制日期和文件路径格式 ([#34118](https://github.com/neovim/neovim/issues/34118))：增强了 LSP 日志的可读性和自定义性
+
+这些改进使 Neovim 的 LSP 客户端更加强大和灵活，特别是对于大型项目和 AI 辅助开发场景。
```
nit: 这种尚未合并的 PR 移到社区咨询那里可能更好？这样开发近况这里确保都是已经合并的

v1nh1shungry commented on posts/013.md:
```diff
@@ -0,0 +1,124 @@
+
+### 终端功能稳定性增强
+
+[PR #34234](https://github.com/neovim/neovim/pull/34234) 修复了终端模式下处理过多 CSI 参数时可能导致的栈溢出问题。这一修复提高了终端模式的稳定性和可靠性，特别是在处理复杂终端输出时。
```
nit: 这里链接的 PR 是 backport 的，可能链接到原始 PR 更好？

v1nh1shungry reviewed (approved):
LGTM!

jinzhongjia committed:
fix(posts): 修正PR链接编号

- 将PR #34234更正为PR #34012

v1nh1shungry commented on posts/013.md:
```diff
@@ -0,0 +1,124 @@
+### 文档系统重组
+
+本周最显著的变化之一是对文档系统的重要重组。
+[PR #34212](https://github.com/neovim/neovim/pull/34212)这项改进由 [@brianhuster](https://github.com/brianhuster) 完成：
```
一个小疑问：这里用 `@` 对方会不会收到通知？特别是考虑到 Release Notes？

jinzhongjia committed:
fix(posts/013): 修正 PR 链接和作者引用错误
- 更正部分 PR 链接编号
- 统一作者引用格式，去除多余 @ 符号
- 修复 Tree-sitter、API、显示和构建系统相关条目的 PR 链接错误
